ShareCar.Client/src/components/Driver/DriverMap.jsx

```diff
@@ -276,14 +244,21 @@ console.log(indexas);
   setInputFrom(value) {
     var inputField = document.querySelector("#driver-address-input-from");
     inputField.value = value;
-    this.state.route.fromAddress = value;
+
+    var routeState = this.state.route;
+    routeState.fromAddress = value;
+    this.setState({ route: routeState });
```

Comment:
State privalo būti immutable, bet šitoj vietoj keiti state'ą. Geriau rašyti this.setState({ ...this.state.route, fromAddress: value}), taip bus sukurta kopija su nauja reikšme.